app/src/org/commcare/models/database/user/UserDatabaseUpgrader.java

```diff
@@ -731,6 +737,21 @@ private boolean upgradeTwentyFiveTwentySix(SQLiteDatabase db) {
         }
     }
 
+    private boolean updateTwentySixTwentySeven(SQLiteDatabase db) {
```

Comment:
all the previous references to `FormRecord` in this class should now reference `FormRecordV5` , [example](https://github.com/dimagi/commcare-android/blob/master/app/src/org/commcare/models/database/user/UserDatabaseUpgrader.java#L681).